### ISSUE TITLE
Public Key Export contains text "UserID" instead of really userid

### ISSUE THREAD

**Issue by martin-dufka** [ProtonMail/WebClient#20]
Settings->security->key export->download PublicKey

Public Key file contains text "UserID" instead of really userid (DISPLAY_NAME <email address>)

**Comment by bartbutler** [comment]
Yep, we are aware of this. That's one reason why this is 'experimental'--the some keys have issues are aren't really ready to be exported. We'll address this in an update, hopefully soon.
